Resources/views/macros.html.twig

```diff
@@ -1,38 +1,34 @@
 {#
 Example:
-
 {% import "A2lixTranslationFormBundle::macros.html.twig" as a2lixTranslations %}
-
-{{ a2lixTranslations.partialTranslations(editForm.translations, ['title','description']) }}
-{{ a2lixTranslations.partialTranslations(editForm.translations, ['url']) }}
+{{ a2lixTranslations.partialTranslations(editForm.translations, ['title','description'], 'my_block') }}
+{{ a2lixTranslations.partialTranslations(editForm.translations, ['url'], 'my_other_html_block') }}
 #}
-
-{% macro partialTranslations(form, fieldsNames) %}
+{% macro partial(form, fieldsNames, block) %}
```

Comment:
This is a BC Break currently for those who already use this macro without block.

So it will need some changes, like add on next line:

    {% set block = block ? block ~'-' : '' %}

and adapt L13 & L25